extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts

```diff
@@ -350,8 +350,7 @@ export class MiaaComputeAndStoragePage extends DashboardPage {
 			currentCPUSize = '';
 		}
 
-		this.coresRequestBox!.placeHolder = currentCPUSize;
-		this.coresRequestBox!.value = '';
+		this.coresRequestBox!.value = currentCPUSize;
```

Comment:
please search and see whether you have other places in arc extension having such problem.

Reply:
I believe it behaves the same way in postgres. I'll submit a separate PR for that